### ISSUE TITLE
Splitter Distance Error

### ISSUE THREAD

**Issue by yinnyann** [SubtitleEdit/subtitleedit#2533]
Error Message: SplitterDistance must stay between Panel1MinSize and Width - Panel2MinSize

![splitterdistance_error](https://user-images.githubusercontent.com/25672350/29762202-9d0be5b8-8ba3-11e7-9152-07f0d1ce75a6.png)

**Comment by niksedk** [comment]
How can I re-create this?

**Comment by yinnyann** [comment]
Hi Nikolaj,

I don't really know what happened. I installed the software some days ago.
It happened one of the first times when I opened it.

It only happened once, no more issue.
I am now working normally with it.

Regards and thanks for this great and easy-to-use soft.

Yann
www.yinnyann.com

Le 31 août 2017 8:39 AM, "Nikolaj Olsson" <notifications@github.com> a
écrit :

How can I re-create this?

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/SubtitleEdit/subtitleedit/issues/2533#issuecomment-326204395>,
or mute the thread
